vhdl/vgen_process_simulink_model.m

```diff
@@ -98,7 +98,7 @@
 disp(['      created tcl file: ' outfile])
 
 disp('vgen: Executing Quartus workflow')
-vgenQuartus(infile, hdlpath + "/quartus/")
+%vgenQuartus(infile, hdlpath + "/quartus/")
```

Comment:
I imagine you just commented this out to make testing faster?

Reply:
Ah yes, indeed that is why I commented it out. I definitely feel like we should make add some configuration so it can just be opted out of. I don't see it being overly needed by end users but it avoids issues like this from reoccurring.

Reply:
I’m definitely in favor of adding easy configuration options.